datastore/error.js

```diff
@@ -13,18 +13,20 @@
 
 'use strict';
 
-// By default, gcloud will authenticate using the service account file specified
-// by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use the
-// project specified by the GCLOUD_PROJECT environment variable. See
-// https://googlecloudplatform.github.io/gcloud-node/#/docs/guides/authentication
-var gcloud = require('gcloud');
-var dataset = gcloud.datastore();
+// By default, the client will authenticate using the service account file
+// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
+// the project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
+var Datastore = require('@google-cloud/datastore');
+
+// Instantiate a datastore client
+var datastore = Datastore();
 
 // [START error]
 function runQuery (cb) {
-  var query = dataset.createQuery(['Company']).start('badrequest');
+  var query = datastore.createQuery(['Company']).start('badrequest');
```

Comment:
No, chaining calls is definitely a common Node.js thing, especially around streams and functional style programming, and apparently with the Datastore query object.